package.json

```diff
@@ -1,12 +1,39 @@
 {
-  "name": "webpacker",
-  "version": "1.0.0",
+  "name": "@rails/webpacker",
+  "version": "2.0.0",
   "description": "Webpacker makes it easy to use the JavaScript preprocessor and bundler [Webpack](https://webpack.github.io) to manage application-like JavaScript in Rails. It coexists with the asset pipeline, as the purpose is only to use Webpack for app-like JavaScript, not images, css, or even JavaScript Sprinkles (that all continues to live in app/assets).",
-  "main": "index.js",
+  "main": "package/index.js",
```

Comment:
Need to add files we are distributing from NPM so the package doesn't include ruby files. 

```
  "files": [
    "package"
  ]
```

Reply:
👍 e0ba006f64c5f4e1f871a8bd23817c65004b3583